loadtest/store/memstore/store_test.go

```diff
@@ -107,4 +119,26 @@ func TestChannelMembers(t *testing.T) {
 		require.NoError(t, err)
 		require.Equal(t, &expected, member)
 	})
+
+	t.Run("Remove channel members", func(t *testing.T) {
+		s := New()
+		channel := &model.Channel{Id: model.NewId()}
+		err := s.SetChannel(channel)
+		require.NoError(t, err)
+		channelMember1 := model.ChannelMember{
+			ChannelId: channel.Id,
+			UserId:    model.NewId(),
+		}
+		channelMember2 := model.ChannelMember{
+			ChannelId: channel.Id,
+			UserId:    model.NewId(),
+		}
+		channelMembers := append(model.ChannelMembers{}, channelMember1, channelMember2)
+		err = s.SetChannelMembers(channel.Id, &channelMembers)
+		require.NoError(t, err)
+		require.Equal(t, 2, len(s.channelMembers[channel.Id]))
+		err = s.RemoveChannelMember(channel.Id, channelMember1.UserId)
+		require.NoError(t, err)
+		require.Equal(t, 1, len(s.channelMembers[channel.Id]))
```

Comment:
I think it's safest to check that the result is exactly what we expect and not just the change in length.